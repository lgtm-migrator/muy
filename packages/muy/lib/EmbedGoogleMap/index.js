import React from "react"
import Iframe from "../Iframe"

const EmbedGoogleMap = ({ apiKey, coordinates, ...props }) => (
  <Iframe
    allowFullScreen={true}
    src={`https://www.google.com/maps/embed/v1/place?q=${coordinates}&key=${apiKey}`}
    {...props}
  />
)

export default EmbedGoogleMap
