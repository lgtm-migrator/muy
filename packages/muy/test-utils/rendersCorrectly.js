import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
