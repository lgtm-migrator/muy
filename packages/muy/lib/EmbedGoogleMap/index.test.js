import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import EmbedGoogleMap from "."

describe("<EmbedGoogleMap /> field", () => {
  it("should render correctly", () => {
    rendersCorrectly(<EmbedGoogleMap />)
  })
})
