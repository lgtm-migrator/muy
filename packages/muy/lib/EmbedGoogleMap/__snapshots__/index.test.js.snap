// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmbedGoogleMap /> field should render correctly 1`] = `
<iframe
  allowFullScreen={true}
  className="MuiBox-root MuiBox-root-2"
  frameBorder={0}
  src="https://www.google.com/maps/embed/v1/place?q=undefined&key=undefined"
/>
`;
